tests/testflows/docker-compose/.config/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG grafana_version=latest
```

Comment:
use original /.config/
do not copy files
remove all  testflows/docker-compose/.config